include/cpuinfo.h

```diff
@@ -593,6 +594,21 @@ enum cpuinfo_uarch {
 
 	/** HiSilicon TaiShan v110 (Huawei Kunpeng 920 series processors). */
 	cpuinfo_uarch_taishan_v110 = 0x00C00100,
+
+	/** POWER 7. */
```

Comment:
Please follow the pattern VendorName ArchName
```suggestion
	/** IBM POWER 7. */
```

Reply:
I have followed suggested change and followed the pattern VendorName ArchName